UsbIpServer/RegistryUtils.cs

```diff
@@ -171,10 +173,16 @@ public static void SetDeviceAddress(ExportedDevice device, string address)
             var key = GetRegistryKey(device);
             if (key != null)
             {
-                key.SetValue("IPAddress", address);
+                key.SetValue(IPAddressName, address);
```

Comment:
I guess this should really go below the "Attached" volatile subkey. I'll move it later.